src/sentry/replays/endpoints/project_replay_breadcrumbs.py

```diff
@@ -0,0 +1,246 @@
+import uuid
+from collections.abc import Callable, MutableMapping, Sequence
+from typing import TypedDict
+
+from drf_spectacular.utils import extend_schema
+from rest_framework.request import Request
+from rest_framework.response import Response
+from snuba_sdk import Column, Condition, Entity, Limit, Offset, Op, Query
+
+from sentry import features
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.models.project import Project
+from sentry.replays.lib.eap.read import query
+from sentry.replays.lib.eap.snuba_transpiler import RequestMeta, Settings
+from sentry.utils.cursors import Cursor, CursorResult
+
+QUERY_SETTINGS: Settings = {
+    "attribute_types": {
+        # Common
+        "replay_id": str,
+        "category": str,
```

Comment:
i believe in the sdk we use `category` for some things, and `op` for others.
examples: `category: 'ui.click'` & `op: 'resource.fetch',`

it's common enough that there is a function `getFrameOpOrCategory()`
is `category` the field to use for searching if we want either type of thing? cuz that's fine.

Reply:
Yeah I'm just using category.  It simplifies querying to have to only target one field type.  In general the storage format can't/won't match the breadcrumb format for a variety of reasons but I can reconstruct the breadcrumb format manually when we serialize (to the best of ability -- lots of fields already dropped at this stage).